corehq/motech/rate_limiter.py

```diff
@@ -28,7 +28,7 @@ def _get_per_user_repeater_wait_milliseconds_rate_definition(domain):
                 per_hour=30,
                 per_minute=0.6,
                 per_second=None,
-            ),
+            ).times(1000),
```

Comment:
nit: this seems like _the way_ to set the units for a RateDefinition class, but I kinda like the idea of just passing in the units as something string constants or something (seconds, milliseconds, etc). Though again understand that this is outside of the scope of this change.

Reply:
Yeah I thought of using a named constant like `MILLISECONDS = 1000` or something, but then I thought that was even more confusing, so I just did this.